src/04-modality-specific-files/29-motion.md

```diff
@@ -0,0 +1,224 @@
+| ------------ | --------------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| TaskName     | REQUIRED              | string        | Name of the task. No two tasks should have the same name. The task label included in the file name is derived from this TaskName field by removing all non-alphanumeric `[a-zA-Z0-9]` characters. |
+
+Generic fields (shared with other BIDS modalities) SHOULD be present: For consistency between studies and institutions, we
+encourage users to extract the values of these fields from the actual raw data.
+Whenever possible, please avoid using ad hoc wording.
+
+| **Key name**                | **Requirement level** | **Data type** | **Description**                                                                                     |
+| --------------------------- | --------------------- | ------------- | --------------------------------------------------------------------------------------------------- |
+| InstitutionAddress          | RECOMMENDED           | string        | The address of the institution in charge of the equipment that produced the composite instances.    |
+| InstitutionalDepartmentName | RECOMMENDED           | string        | The department in the institution in charge of the equipment that produced the composite instances. |
+| InstitutionName             | RECOMMENDED           | string        | The name of the institution in charge of the equipment that produced the composite instances.       |
+| Instructions                | RECOMMENDED           | string        | Text of the instructions given to participants before the recording.                                |
+| TaskDescription             | RECOMMENDED           | string        | Longer description of the task.                                                                     |
+
+Specific MOTION fields MUST be present:
+
+| **Key name**       | **Requirement level** | **Data type** | **Description**                                                                                                       |
+| ------------------ | --------------------- | ------------- | --------------------------------------------------------------------------------------------------------------------- |
+| SamplingFrequency  | REQUIRED              | number        | Nominal sampling rate of the tracking system in Hz.                                                                   |
+| TrackingSystemName | REQUIRED              | string        | Name of the tracking system. The value should match the `tracksys` value of the corresponding *_motion.tsv file name. |
```

Comment:
How is this field `TrackingSystemName`different form the general `Manufacturer` and `ManufacturersModelName`?

Reply:
The field is intended as a value for the 'tracksys' entity used in the filename of *motion.tsv and *motion.json files. (for example the raw motion data is stored under 'sub-001_task-turning_tracksys-imu_motion.tsv'. TrackingSystemName in this case is 'imu'.

Reply:
That would be the company's name (e.g. BWSensing) and the model name of a system (BW-IMU600).